packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/NextButton/NextButton.js

```diff
@@ -1,98 +1,50 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import styled, { ThemeProvider } from 'styled-components'
-import theme from 'styled-theming'
-import { adjustHue } from 'polished'
 import counterpart from 'counterpart'
-import { Button, Text } from 'grommet'
-import { FormNextLink } from 'grommet-icons'
-import zooTheme from '@zooniverse/grommet-theme'
+import { Button, Grommet, Text } from 'grommet'
+import { merge } from 'lodash'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { withTheme } from 'styled-components'
+
+import nextButtonTheme from './theme'
 import en from './locales/en'
 
 counterpart.registerTranslations('en', en)
 
-const gold = zooTheme.global.colors['neutral-4']
-
-// TODO: box-shadow, border-radius, and disabled likely can move into the grommet theme.
-export const StyledNextButton = styled(Button)`
-  background: ${theme('mode', {
-    dark: zooTheme.global.colors['dark-1'],
-    light: gold
-  })};
-  border: ${theme('mode', {
-    dark: `solid thin ${gold}`,
-    light: `solid thin ${gold}`
-  })};
-  border-radius: 0;
-  box-shadow: none;
-  color: ${theme('mode', {
-    dark: gold,
-    light: 'black'
-  })};
-  flex: 3 0 auto;
-  padding: 0.5em;
-  text-transform: capitalize;
-
-  svg {
-    fill: ${theme('mode', {
-    dark: gold,
-    light: 'black'
-  })};
-    stroke: ${theme('mode', {
-    dark: gold,
-    light: 'black'
-  })};
-  }
-
-  &:hover:not(:disabled), &:focus:not(:disabled) {
-    background: ${theme('mode', {
-    dark: gold,
-    light: adjustHue(-7, gold)
-  })};
-    border: ${theme('mode', {
-    dark: `solid thin ${gold}`,
-    light: `solid thin ${adjustHue(-7, gold)}`
-  })};
-    box-shadow: none;
-    color: ${theme('mode', {
-    dark: 'black',
-    light: 'black'
-  })};
-  }
-
-  &:disabled {
-    cursor: not-allowed;
-  }
-  `
+const Label = (
+  <Text size='medium'>
+    {counterpart('NextButton.next')} →
+  </Text>
+)
 
-function NextButton ({ autoFocus, disabled, classifierTheme, onClick }) {
+function NextButton (props) {
+  const { autoFocus, disabled, onClick, theme } = props
+  const mergedThemes = merge({}, theme, nextButtonTheme)
   return (
-    <ThemeProvider theme={{ mode: classifierTheme }}>
-      <StyledNextButton
+    <Grommet theme={mergedThemes}>
```

Comment:
I was looking over the Grommet docs again today and maybe we should be using `ThemeContext` instead of the Grommet component instead to be doing stuff like this? https://v2.grommet.io/themecontext

Reply:
Done :+1: